.gitignore

```diff
@@ -6,4 +6,8 @@ venv
 *.iso
 
 .DS_Store
-*/.DS_Store
\ No newline at end of file
+*/.DS_Store
+
+# These are files that get built by make and should not be checked in to the repo
+healthcheck.py
+vrnetlab.py
```

Comment:
won't these changes have a side effect of you not being able to check in changes to `common/vrnetlab.py[healthcheck.py]`?

Reply:
Since they're already tracked, it still keeps them in git and allows them to be modified (at least from my testing). It would be better to explicitly allow common/vrnetlab.py and common/healthcheck.py. I'll make that change